### ISSUE TITLE
Multibit wallet repository question

### ISSUE THREAD

**Issue by ABISprotocol** [ABISprotocol/ABIS#3]
The multibit wallet for ABIS is currently at
https://github.com/bitwiseban/ABIS-demo-wallet

The question I'm asking myself is what would be the best way to place a copy of that in the ABISprotocol / ABIS repository for people to work on directly from this repository?

Is there a way to simplify this by setting up a github organization (suggested by another issue in this repo) or could it be done without the github organization being set up?  Presumably so, but curious what the best way would be.

**Comment by ktorn** [comment]
Do not work on that fork. It is obsolete Multibit code and any new development should fork from the current Multibit HD codebase: https://github.com/bitcoin-solutions/multibit-hd

**Comment by ABISprotocol** [comment]
Oh, well ok. Well, shoot.  Is there another that has the current codebase?  Realizing that https://github.com/bitwiseban/ABIS-demo-wallet now has obsolete code... What can be done to save the work that you did on that and include the current Multibit HD codebase?

**Comment by ktorn** [comment]
There isn't a great deal of code to be saved.

I only added 22 lines of code in a single commit, to create a "donations" menu-item:
https://github.com/bitwiseban/ABIS-demo-wallet/commit/fd4e7165c0c8f422d3836018faebad08999be4ca

That code can be easily re-done once the new Multibit HD fork is in place.

**Comment by ABISprotocol** [comment]
@ktorn I realize you are now devoting attentions fully on VDP and subsatoshi, and thanks for this response.  Essentially, this would involve me forking the Multibit HD - this - 
https://github.com/jim618/multibit
so that it would be in the ABIS repository,
and then adding in the lines of code which are currently shown here
https://github.com/bitwiseban/ABIS-demo-wallet/commit/fd4e7165c0c8f422d3836018faebad08999be4ca
then moving on from there?
(By that point I will already have finished the github organization, so I'm assuming I will be able to add the resulting repository to show as part of what people see when they examine the github org.)
Thanks for your help and clarification.

**Comment by ktorn** [comment]
"current Multibit HD codebase: https://github.com/bitcoin-solutions/multibit-hd"

**Comment by ABISprotocol** [comment]
@ktorn OK, thanks!
